### PR TITLE
[WIP] version 0.6 release

### PR DESCRIPTION
**Pull request by robintibor**
I think now it is ready for next release, except waiting for one bugfix from  @gemeinl next week. please tell if there is any objections or anything needed to be done

### CODE REVIEW

**Comment by cedricrommel** [comment]
Really sorry @robintibor, but I found a last bug running my tests with a different random seed this morning 😬 
https://github.com/braindecode/braindecode/pull/339

**Comment by cedricrommel** [comment]
Any news on the release @robintibor ? :) Are you waiting for some other PRs to be merged before?

**Comment by robintibor** [comment]
Waited for https://github.com/braindecode/braindecode/pull/341, but now it's merged, so from my side we could release any day now. E.g. Tomorrow? Monday? @hubertjb @gemeinl @agramfort @sliwy ?

**Comment by agramfort** [comment]
whenever you're ready

can you share a draft of the what's new section for the new release?

**Comment by robintibor** [comment]
How do you mean exactly? in https://github.com/braindecode/braindecode/blob/e15c863f9326437059ddab218f29e4ee6604cb62/docs/whats_new.rst we should have now collected all the whats_news already during the PR process.... or did you have something more in mind? @agramfort

**Comment by agramfort** [comment]
do we have a rendered artifact to see if it renders fine? any doc
improvement we want to do before the release?

>

**Comment by robintibor** [comment]
Would need to merge https://github.com/braindecode/braindecode/pull/366 first to make everything run again

File: docs/whats_new.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ What's new
 
 .. _current:
 
-Current (0.5.2.dev0)
+Version 0.6 (2021-12-06)
 --------------------
```
Comment (agramfort) [inline docs/whats_new.rst:22]:
```suggestion
------------------------
```

otherwise RST does not render well

**Comment by robintibor** [comment]
Merged and release now also on PyPI https://pypi.org/project/Braindecode/

**Pull request merged**